src/main/java/io/chrislowe/discordle/game/GameManager.java

```diff
@@ -32,6 +32,10 @@ public void startNewGame() {
         gameIsActive = true;
     }
 
+    public Game getGame() {
```

Comment:
If practical, I would prefer if we can keep the encapsulation of the Game object inside of GameManager. Do you think we can write this similarly to #getWordGuesses which pulls directly from the internally managed game object?

Reply:
Done!